tools/atomic_wget.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+TARGET_FILE=$(basename "$1")
+TEMP_FILE="/tmp/$TARGET_FILE"
+
+# Check if $3 is provided and not empty
+if [ -n "$3" ]; then
+    WGET_OPTIONS="$3"
+else
+    WGET_OPTIONS=""
+fi
+
+wget --continue $WGET_OPTIONS -O "$TEMP_FILE" "$2" || { \
```

Comment:
I don't think `--continue` will have any effect when `-O` is also given.

Reply:
Not really. I have tried the intermittent transfers before and found that it indeed takes effect. Here is an example.
![image](https://github.com/asterinas/asterinas/assets/24353687/fd7f74d9-3a79-4158-a9b2-32ad28109b57)